### DISCUSSION TITLE
What is the advantage of Libva?

### DISCUSSION THREAD

**Discussion by gniknalu** [roflcoopter/viseron#231] [Q&A] [answered]
```
****** Checking for hardware acceleration platforms ******
OpenCL is available!
VA-API cannot be used
CUDA is available!
```

Is there any advantage to running Libva ?

**Comment by roflcoopter** [comment, marked as answer]
VA-API can be used for hardware accelerated decoding of your camera streams.
Since you are using an NVIDIA GPU it will take care of it instead.

One usecase tho is if you have many cameras your GPU will eventually be "full", then you could use VA-API to decode the remainder of the streams

**Reply by gniknalu** [reply]
Going to give it a shot and add it. 

Thanks again!
